numpy/fft/_pocketfft.pyi

```diff
@@ -25,113 +25,113 @@ _NormKind: TypeAlias = L["backward", "ortho", "forward"] | None
 
 def fft(
     a: ArrayLike,
-    n: int | None = ...,
-    axis: int = ...,
-    norm: _NormKind = ...,
-    out: NDArray[complex128] | None = ...,
+    n: int | None = None,
+    axis: int = -1,
+    norm: _NormKind = None,
```

Comment:
I'm to blame here, but maybe we should also be explicit about the `None` here, like we try to be `DTypeLike | None`. And maybe even remove the `| None` from the alias while we're at it.